### PR TITLE
Support versions of git without --name-only

### LINKED ISSUE #1732: [TASK] Support git 1.8
--------
Issue Description
--------

**Is your feature request related to a problem? Please describe.**
Raven plugin initialization does not work with git 1.8 because it does not have --name-only and so we get errors like:

```
python scripts/install_plugins.py -a
error: unknown option `name-only'
usage: git config [options]

Config file location
    --global              use global config file
    --system              use system config file
    --local               use repository config file
    -f, --file <file>     use given config file
    --blob <blob-id>      read config from given blob object

Action
    --get                 get value: name [value-regex]
    --get-all             get all values: key [value-regex]
    --get-regexp          get values for regexp: name-regex [value-regex]
    --replace-all         replace all matching variables: name value [value_regex]
    --add                 add a new variable: name value
    --unset               remove a variable: name [value-regex]
    --unset-all           remove all matches: name [value-regex]
    --rename-section      rename section: old-name new-name
    --remove-section      remove a section: name
    -l, --list            list all
    -e, --edit            open an editor
    --get-color <slot>    find the color configured: [default]
    --get-colorbool <slot>
                          find the color setting: [stdout-is-tty]

Type
    --bool                value is "true" or "false"
    --int                 value is decimal number
    --bool-or-int         value is --bool or --int
    --path                value is a path (file or directory name)

Other
    -z, --null            terminate values with NUL byte
    --includes            respect include directives on lookup

Initializing plugin "ExamplePlugin"
 ... initializing plugin submodule ExamplePlugin ...
 ... plugin located at "/ibmstor/scratch/moosetest/civet_sawtooth2/build_0/raven/plugins/ExamplePlugin" ...
Installing plugins ...
 ... plugin "ExamplePlugin" path created as "/ibmstor/scratch/moosetest/civet_sawtooth2/build_0/raven/plugins/ExamplePlugin" ...
 ... plugin "ExamplePlugin" informed of RAVEN at "/ibmstor/scratch/moosetest/civet_sawtooth2/build_0/raven/framework"...
 ... plugin "ExamplePlugin" succesfully installed!
 ```

**Describe the solution you'd like**
Support git 1.8

**Describe alternatives you've considered**
Use a newer git.

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or task?
- [x] 2. Is it tagged with a priority: critical, normal or minor?
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by joshua-cogliati-inl**
--------
Pull Request Description
--------
##### What issue does this change request address? 
Closes #1732 

##### What are the significant changes in functionality due to this change request?
Supports an older version of git.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: scripts/install_plugins.py [hunk 1]
```diff
@@ -63,11 +63,11 @@
   owd = os.getcwd()
   cwd = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
   os.chdir(cwd)
-  subsOut = os.popen('git config --file .gitmodules --name-only --get-regexp path').read()
+  subsOut = [x.split(' ')[0] for x in os.popen('git config --file .gitmodules --get-regexp path').read().split("\n")]
```
Comment (wangcj05) [inline scripts/install_plugins.py:66]: Do we need to specify the empty space for the "x.split(' ')" here? Can we just use the default, i.e., "x.split()"?

**Review by wangcj05 — COMMENTED** [review]
@joshua-cogliati-inl I have just one comment for you.

File: scripts/install_plugins.py (hunk 1 above)
Comment (joshua-cogliati-inl) [reply scripts/install_plugins.py:66]: Yes, the default would work, tho' that would split on all whitespace instead of just spaces.

**Review by wangcj05 — APPROVED** [review]

**Comment by wangcj05** [comment]
Tests are green, and checklist is satisfied, PR can be merged. @joshua-cogliati-inl Thanks for fixing the problem

**Pull request merged**